File: easy/n_mod_m/n_mod_m.js
var fs = require('fs');
fs.readFileSync(process.argv[2]).toString().split('\n').forEach(function (line) {
  if (line !== '') {
    nums = line.split(',').map(function (num) { return parseInt(num); });
    console.log(Math.round((nums[0]/nums[1] - Math.floor(nums[0]/nums[1])) * nums[1]));
  }
});

File: easy/n_mod_m/n_mod_m.c
#include <stdio.h>
#include <string.h>
#include <stdlib.h>
#include <math.h>

int main(int argc, char **argv)
{
  FILE *file = fopen(argv[1], "r");
  char line[1024];
  float n,m;

  while (fgets(line, 1024, file)) {
    n = atof(strtok(line, ","));
    m = atof(strtok(NULL, ","));
    printf("%i\n", (int)(((n / m  - floor(n / m)) * m)+0.5));
  }
  return 0;
}
